tests/feature/test_cucumber_json.py

```diff
@@ -31,6 +31,14 @@ def __cmp__(self, other):
 
 def test_step_trace(testdir):
     """Test step trace."""
+    testdir.makefile(".ini", pytest=textwrap.dedent("""
+    [pytest]
+    markers =
+        scenario-passing-tag
+        scenario-failing-tag
+        scenario-outline-passing-tag
+        feature-tag
```

Comment:
Why do we need those markers here? Is it documented somewhere?

Reply:
They are used by the feature file of line 42. Pytest 4.5+ emits warnings if you se markers without defining them in pytest.ini, and we treat warnings as errors in our test configuration.